gson/src/main/java/com/google/gson/Gson.java

```diff
@@ -1381,11 +1374,9 @@ private TypeAdapter<T> delegate() {
 
   @Override
   public String toString() {
-    return new StringBuilder("{serializeNulls:")
-        .append(serializeNulls)
-        .append(",factories:").append(factories)
-        .append(",instanceCreators:").append(constructorConstructor)
-        .append("}")
-        .toString();
+    return "{serializeNulls:" + serializeNulls
+      + ",factories:" + factories
```

Comment:
These continuation lines should be indented +4 rather than +2. You might find [google-java-format](https://github.com/google/google-java-format) useful, though you probably only want to run it on changed lines rather than the whole file.